dependencies.yaml

```diff
@@ -20,6 +20,7 @@ dependencies:
       - output_types: [conda, requirements]
         packages:
           - cmake>=3.23.1
+          - ninja
```

Comment:
```suggestion
          - ninja
          - make
```

Reply:
added in https://github.com/rapidsai/rapids-cmake/actions/runs/3431545485/jobs/5719970033